packages/bumbag/src/Group/styles.ts

```diff
@@ -29,16 +34,74 @@ export const Group = (styleProps) => cssClass`
     position: relative;
   }
 
+  & > *:first-child:nth-last-child(-n+1) {
```

Comment:
I was just thinking we could possible just flip the roles of `:first-child` and `:last-child` instead of adding more selectors?

At the moment, we remove the border radius from the first and last child elements, however, what if we applied the border radii instead? Then remove the border-radius on the `& > *` selector?

Reply:
You mean that the border radii will only be applied to first and last childs?
I think this is even a better idea because this makes the code cleaner as you said.

Should I change it in this PR or close it and you'll implement it in the main branch?

Reply:
Go for it

Reply:
I am a bit unsure about the changes I made now.
Removed the selectors introduced by this PR and replaces all `border-...-radius: 0;` with the helper function `border-...-radius: ${borderRadius(
          styleProps.borderRadius,
          styleProps.borderRadius
        )(styleProps)};`.

Can you please review the changes?
I'm unsure about my usage of `borderRadius()` function and can't test at the moment.